Drawer/Drawer.js

```diff
@@ -0,0 +1,84 @@
+import compose from 'ramda/src/compose';
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Slottable from '@enact/ui/Slottable';
+import Transition from '@enact/ui/Transition';
+
+import Skinnable from '../Skinnable';
+// import Icon from '../Icon';
+
+import PopupState from '../Popup/PopupState';
+
+import componentCss from './Drawer.less';
+
+const DrawerBase = kind({
+	name: 'Drawer',
+
+	propTypes: {
+		css: PropTypes.object,
+		footer: PropTypes.node,
+		header: PropTypes.node,
+		noAnimation: PropTypes.bool,
+		onHide: PropTypes.func,
+		open: PropTypes.bool
+	},
+
+	defaultProps: {
+		noAnimation: false,
+		open: false,
+		orientation: 'vertical'
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'drawer'
+	},
+
+	render: ({footer, children, css, noAnimation, onHide, open, header, ...rest}) => {
+		return (
+			<Transition
+				noAnimation={noAnimation}
+				visible={open}
+				direction="left"
+				duration="short"
+				type="slide"
+				className={css.drawerTransitionContainer}
+				onHide={onHide}
+				css={css}
+			>
+				<Layout
+					{...rest}
+				>
+					{/* <Icon small onClick={onClose} className={css.closeButton}>closex</Icon>*/}
```

Comment:
What are your plans for this (cruft/future)? I assume this was/will-be a built-in feature to allow managing closed state. I'd generally suggest removing comments, but if we're wanting to keep this for a future feature, then perhaps we could also:
- comment out the `css.closeButton` styles in `Drawer.less` for now
- create/destructure and comment out the prop for `onClose`, since it's used without reference here (even though it's commented).

Reply:
Those are good points. I had indeed left this in for future consideration, something to the similar visual effect of the old Moonstone drawer, with the close button looking like a little tab in the center. Commenting out the other parts and including a forward-referencing comment/description explaining the intent is a great idea. I'll add that and make this aspect look a little more presentable. The locally controlled closing ability wasn't immediately needed for this scenario, but will surely be in the future.